distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -19,8 +19,12 @@ deny {
     permission java.security.SecurityPermission "createAccessControlContext";
     permission java.security.SecurityPermission "setPolicy";
     permission java.security.SecurityPermission "insertProvider";
+    permission java.security.SecurityPermission "insertProvider.*";
+    permission java.security.SecurityPermission "removeProvider";
     permission java.security.SecurityPermission "removeProvider.*";
+    permission java.security.SecurityPermission "clearProviderProperties";
```

Comment:
❓  Why do we need this variation of the permission? It doesn't look like it's listed as a SecurityPermission. (neither does line 27 the removeProviderProperty one) https://docs.oracle.com/javase/8/docs/api/java/security/SecurityPermission.html

Reply:
https://docs.oracle.com/javase/8/docs/technotes/guides/security/permissions.html#SecurityPermission

Reply:
It is not in there that I see. neither is `removeProviderProperty`

Reply:
Search the page, please.

Reply:
It is not there, there is only this form `removeProviderProperty.{provider name}` , which from what I can tell is not the same as what is listed above

Reply:
They explicitly list when something has a `thing.name` vs just a `thing`, unless this is some implicit secret security policy trick that they don't list in their permissions?

I don't see anything listed here regarding any special forms that would allow this here:
http://docs.oracle.com/javase/8/docs/technotes/guides/security/PolicyFiles.html#FileSyntax

Reply:
I'll look again in the morning when I'm not doing this on my phone.

Reply:
And now is when I apologize for trying to look at this on my phone. You are absolutely correct, @rzwiefel; these additions are incorrect permissions in some cases and do not make sense in others. E.g. there is no permission `clearProviderProperties` without a qualifying provider name; and "the `insertProvider` permission subsumes the `insertProvider.{provider name}` permission".

Reply:
Yeah I got a little carried away there, I'll clean it up.